unit_tests/test_tlslite_messages.py

```diff
@@ -489,7 +489,8 @@ def test___repr__(self):
                 "random=bytearray(b'\\x00'), session_id=bytearray(b''), "\
                 "cipher_suites=[], compression_methods=[0], "\
                 "extensions=[TLSExtension(extType=0, "\
-                "extData=bytearray(b''), serverType=False)])",
+                "extData=bytearray(b''), serverType=False, "
+                "encExtType=False)])",
                 repr(client_hello))
 
     def test_getExtension(self):
```

Comment:
To my understanding this doesn't reject plain extensions that should have been encrypted, though it covers the description above.

r+